### ISSUE TITLE
remove locatives (basic or distributive) from paradigms (for now)

### ISSUE THREAD

**Issue by dwhieb** [UAlbertaALTLab/morphodict#949]
**tl;dr:** We should remove the distributive locative from the noun paradigms in itwêwina.

---

# Details

There are two locative endings in Plains Cree:

* _-ihk_: basic locative, 'in, on'
* _-inâhk_: distributive locative, 'among, in the territory of, an abundance of'

These locatives can be added to the ends of nouns, but not all nouns. For most nouns it's semantically incoherent to add them, or simply not a recognized word.

The Plains Cree FST recognizes and generates locative forms of all nouns (which is good).

However, itwêwina also displays the distributive locative form of all nouns (which is bad). Not all nouns should take the distributive locative.

Until we have a system in place for indicating which specific nouns take the basic locative or distributive locative, Arok would like to remove both locatives from the paradigm tables in itwêwina.
